Documentation/concepts/failure_behavior.rst

```diff
@@ -0,0 +1,37 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************
+Failure Behavior
+****************
+
+If Cilium loses connectivity with the KV-Store, it guarantees that:
+
+* Normal networking operations will continue;
+
+* If policy enforcement is enabled, the existing `endpoints` will still have
+  their policy enforced but you will lose the ability to add additional
+  containers that belong to security identities which are unknown on the node;
+
+* If services are enabled, you will lose the ability to add additional services
```

Comment:
Why? The allocation is local.

Reply:
This is existing documentation. We can fix it up in a separate PR. This PR Is only splitting `concepts.rst` into individual files.